.cirrus.yml

```
@@ -40,15 +40,15 @@ task:
   name: tests-macos-0
   use_compute_credits: $CIRRUS_USER_COLLABORATOR == 'true' && $CIRRUS_PR == ''
   script: scripts/verify_tests_on_master.sh --shards 2 --shard-index 0
-  osx_instance:
-    image: mojave-base
+  macos_instance:
+    image: ghcr.io/cirruslabs/macos-ventura-base:latest
```

This should be pinned to a specific Xcode version to minimize out-of-band failures.
https://github.com/flutter/plugins/blob/320461910156084cf88aaceb7873c958b455ba7d/.cirrus.yml#L38

```suggestion
    image: ghcr.io/cirruslabs/macos-ventura-xcode:14
```

---

done